.github/workflows/curator-api-node.yml

```diff
@@ -33,3 +33,8 @@ jobs:
         working-directory: verification/curator-service/api
         env:
           CI: true
+      - name: Upload coverage
+        uses: codecov/codecov-action@v1
+        with:
+          file: verification/curator-service/api/coverage/coverage-final.json
+          fail_ci_if_error: true
```

Comment:
Is there an override option to be able to merge something without coverage if necessary?

Reply:
github doesn't enforce CI to pass to be able to merge, you can --force anytime.
This will make the CI fail if the upload to codecov fails which I think is reasonable
https://github.com/marketplace/actions/codecov

Reply:
Got it, thanks.